indico/modules/categories/client/js/management.js

```diff
@@ -248,6 +253,8 @@
   };
 
   global.setupCategoryEventList = function setupCategoryEventList(categoryId) {
+    let isEverythingSelected = false;
```

Comment:
Yeah this was just to make eslint happy since it couldn't auto-fix the old `var` (which gets hoisted), and without assigning something here it suggested to make it `const` for some reason (even though it does get reassigned below)...